lib/matplotlib/pyplot.py

```diff
@@ -2247,7 +2247,8 @@ def getname_val(identifier):
 # requested, ignore rcParams['backend'] and force selection of a backend that
 # is compatible with the current running interactive framework.
 if (rcParams["backend_fallback"]
-        and dict.__getitem__(rcParams, "backend") in _interactive_bk
+        and dict.__getitem__(rcParams, "backend") in [
+            k for k in _interactive_bk if k not in ['WebAgg', 'nbAgg']]
```

Comment:
It seems simpler to do:
```suggestion
         and dict.__getitem__(rcParams, "backend") in _interactive_bk
         and dict.__getitem__(rcParams, "backend") not in ['WebAgg', 'nbAgg']
```
or maybe:
```suggestion
       and dict.__getitem__(rcParams, "backend") in (set(_interactive_bk) - {'WebAgg', 'nbAgg'})
```